src/main/java/training/ModuleConstants.java

```diff
@@ -0,0 +1,12 @@
+package training;
+
+import com.ctre.phoenix6.configs.Slot0Configs;
+
+public class ModuleConstants {
+	public static final double ANGULAR_TOLERANCE = 1.8 / 360;
```

Comment:
look into the class `Rotation2d` and use it when using angles

Reply:
consider changing "look into" to more descriptive name